freezeyt.py

```diff
@@ -37,6 +37,9 @@ def start_response(status, headers):
         with open(path / link.lstrip('/'), "wb") as f:
             for item in result:
                 f.write(item)
+            
```

Comment:
Na tomto riadku mas nejake "trailing whitespaces", ktore su vo vseobecnosti v kode neziaduce, vo vacsine pripadov to nie je problem, ale je dobrym zvykom si na to davat pozor (mozno aj tvoj editor ma nejake nastavenie na vymazavanie trailing whitespaces pri ulozeni)

Reply:
vsechna formatovani jsem prave vypnul takze to spise bude ze jsem tam omylem udelal jeden navic a historicky mi to hlidal vzdy auto formatter tak jsem ted na to zapomnel nebo spis nevedel ze je nezadouci